docs/pages/modules/android-lifecycle-listeners.md

```diff
@@ -4,15 +4,21 @@ title: Android Lifecycle Listeners
 
 import { Tab, Tabs } from '~/components/plugins/Tabs';
 
-Some native modules need to apply changes in the **MainActivity.java** or **MainApplication.java** to finish the setup. The Expo Modules system has an extensible design allowing your module to hook into `Activity` or `Application` functions.
+In order to respond to certain Android system events relevant to an app, such as inbound links and configuration changes, it is necessary to override the corresponding lifecycle callbacks in **MainActivity.java** and/or **MainApplication.java**.
+
+The React Native module API does not provide any mechanism to hook into these, and so setup instructions for React Native libraries often include steps to copy code into these files. To simplify and automate setup and maintenance, the Expo module API provides a mechanism that allows your library to hook into `Activity` or `Application` functions.
 
 ## Get Started
 
-You should [create an Android module first](./overview.md#setup), then to create a concrete class that implements the [`Package`](https://github.com/expo/expo/blob/master/packages/expo-modules-core/android/src/main/java/expo/modules/core/interfaces/Package.java) interface. For most cases, you only need to implement `createReactActivityLifecycleListeners` or `createApplicationLifecycleListeners` methods.
+First, you need to have created an Expo module or integrated the Expo modules API in library using the React Native module API. [Learn more](./overview.md#setup)
+
+Inside of your module, a concrete class that implements the [`Package`](https://github.com/expo/expo/blob/master/packages/expo-modules-core/android/src/main/java/expo/modules/core/interfaces/Package.java) interface. For most cases, you only need to implement the `createReactActivityLifecycleListeners` or `createApplicationLifecycleListeners` methods.
 
 ## `Activity` Lifecycle Listeners
 
-If you want to have hook code from `Activity` lifecycles, the `ReactActivityLifecycleListener` is aimed for this use case. `ReactActivityLifecycleListener` hooks into React Native's `ReactActivity` lifecycles by its `ReactActivityDelegate` and provides approximate experience to Android `Activity` lifecycles. We support these lifecycles so far:
```

Comment:
> approximate experience

it may be useful to elaborate on the differences (eg: can't access all lifecycle callbacks, only those that are exposed). anything that we think we will fix that makes this approximate we can list in known issues (like in https://github.com/expo/expo/pull/15261/commits/bbac3f878116dfdb9a936fbee593c9f83bdb3fc4)